src/main/java/com/google/security/zynamics/reil/algorithms/mono2/common/MonoReilSolver.java

```diff
@@ -63,17 +63,17 @@
    * result we skip edges which have not been visited in order to omit artificially inserted entry
    * and exit edges.
    */
-  private final Set<IInstructionGraphEdge> m_traversedEdges = new HashSet<IInstructionGraphEdge>();
+  private final Set<IInstructionGraphEdge> m_traversedEdges = new HashSet<>();
 
   public MonoReilSolver(final IInstructionGraph instructionGraph,
       final AnalysisDirection analysisDirection, final ILattice<LatticeElementType> lattice) {
     m_graph = Preconditions.checkNotNull(instructionGraph,
-        "Error: instruction graph argument can not be null");
```

Comment:
Please leave the Error: prefix in the all the files. These can be used to give unique error numbers to the individual errors making them way more precise. If you want to change the Error message prefix please provide a good justification.